src/components/AboutUs/TimelineSection.js

```diff
@@ -1,15 +1,12 @@
 import React from "react";
-import { makeStyles, StylesProvider } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
 
 import TitleRight from '../../images/about-us/bottom-line/titleRight.svg';
 import TitleLeft from '../../images/about-us/bottom-line/titleLeft.svg';
 import Pin from '../../images/about-us/timeline/pin.svg';
-import Pic1 from '../../images/about-us/timeline/pic1.svg';
-import Pic3 from '../../images/about-us/timeline/pic3.svg';
-import LeftBlob from '../../images/about-us/timeline/leftblob.svg';
-import RightBlob from '../../images/about-us/timeline/rightblob.svg';
-import { DriveEtaOutlined } from "@material-ui/icons";
+import pic1 from '../../images/pngs/pic1.png';
+import pic3 from '../../images/pngs/pic3.png';
```

Comment:
it would be better for us to rename files to what pic they contain (like "Skyline" instead of "pic1")